x-pack/plugins/fleet/server/routes/epm/handlers.ts

```diff
@@ -115,6 +121,51 @@ export const getListHandler: FleetRequestHandler<
   }
 };
 
+export const getInstalledListHandler: FleetRequestHandler<
+  undefined,
+  TypeOf<typeof GetInstalledPackagesRequestSchema.query>
+> = async (context, request, response) => {
+  try {
+    const savedObjectsClient = (await context.fleet).internalSoClient;
+    const res = await getInstalledPackages({
+      savedObjectsClient,
+      ...request.query,
+    });
+
+    const body: GetInstalledPackagesResponse = { ...res };
+
+    return response.ok({
+      body,
+    });
+  } catch (error) {
+    return defaultFleetErrorHandler({ error, response });
+  }
+};
+
+export const getDataStreamsHandler: FleetRequestHandler<
+  undefined,
+  TypeOf<typeof GetDataStreamsRequestSchema.query>
+> = async (context, request, response) => {
+  try {
+    const coreContext = await context.core;
+    const esClient = coreContext.elasticsearch.client.asCurrentUser;
```

Comment:
Could we add a similar comment to [this](https://github.com/elastic/kibana/blob/main/x-pack/plugins/fleet/server/routes/data_streams/handlers.ts#L44) so that it's clear why we use `asCurrentUser`.